doc/Developer-Guide.md

```diff
@@ -42,12 +96,14 @@ ER-Model:
 
 The database consists of just a few entities. The work table contains basic information on imported works and an enabled attribute for controlling access to the files of a work. The tables action_data and action_parameter is for controlling asynchronous actions to be performed on works. The user table contains the login data of the users of the UI. Since the mediaserver UI is strictly for administration, we don't need a user role concept.
 
-The implementation of the database and all database communication uses Spring JPA, all classes are found [here](https://github.com/tuub/kitodo-mediaserver/tree/master/kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/db). 
-The initialization of the database is done using Hibernate, there is no `schema.sql` file. See [InitDbCommand](https://github.com/tuub/kitodo-mediaserver/blob/master/kitodo-mediaserver-cli/src/main/java/org/kitodo/mediaserver/cli/commands/InitDbCommand.java) and [database](https://github.com/tuub/kitodo-mediaserver/tree/master/kitodo-mediaserver-cli/src/main/java/org/kitodo/mediaserver/cli/database).
+The implementation of the database and all database communication uses Spring JPA, all classes are found [here](../kitodo-mediaserver/tree/master/kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/db).
```

Comment:
The path should be `../kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/db`